docs/workflows.rst

```diff
@@ -64,3 +67,59 @@ All the commands should be run on Pulp 3 machine.
             }
         ]
     }
+
+
+Reset migrated Pulp 3 data
+--------------------------
+
+.. note::
+    Nothing is getting reset or modified in any way in Pulp 2.
+
+.. note::
+    Migrated artifacts are untouched. If you are sure, you want to remove them, please run the
+    orphan cleanup task in Pulp 3.
+
+There are cases, when one needs to run a migration from scratch (by default, it's always
+incremental). E.g. some issue happened during the pulp2to3 migration which you are not able to
+recover from.
+
+All the commands should be run on Pulp 3 machine.
+
+
+1. Create a :doc:`Migration Plan <../migration_plan>` which you want to run a migration for.
+
+.. code:: bash
+
+    $ # migrate content for Pulp 2 ISO plugin
+    $ http POST :24817/pulp/api/v3/migration-plans/ plan='{"plugins": [{"type": "iso"}]}'
+
+    HTTP/1.1 201 Created
+    {
+        "pulp_created": "2019-07-23T08:18:12.927007Z",
+        "pulp_href": "/pulp/api/v3/migration-plans/59f8a786-c7d7-4e2b-ad07-701479d403c5/",
+        "plan": "{ "plugins": [{"type": "iso"}]}"
+    }
+
+2. Use the ``pulp_href`` of the created Migration Plan to reset Pulp 3 data
```

Comment:
Maybe be a little more clear about the special endpoint, it's only shown in the script and not mentioned in the instruction.

Reply:
That applies to the ```run/``` endpoint this was copied from as well, I think.